File: Programmers/level2/expressNumber/express_number.py
# 문제 출처 : https://programmers.co.kr/learn/courses/30/lessons/12924

def solution(n):
    answer = 1
    for i in range(1, n//2 + 2):
        plus = i
        for j in range(i + 1, n//2 + 2):
            plus += j
            if plus == n:
                answer += 1
                break
            elif plus > n:
                break
            else:
                continue
    return answer

File: Programmers/level2/expressNumber/expressNumber.js
// https://programmers.co.kr/learn/courses/30/lessons/12924?language=javascript

export const solution = (n) => {
    var answer = 1;
    for(let i = 1; i < parseInt(n/2)+2; i++){
        let sum = i;
        for(let j = i+1; i < parseInt(n/2)+2; j++){
            sum += j;
            if(sum === n){
                answer += 1;
                break;
            }
            if(sum > n){
                break;
            } else {
                continue;
            }
        }
    }
    return answer;
}